src/effects/builtin/reverb/ReverbView.qml

```diff
@@ -1,20 +1,104 @@
 import QtQuick 2.15
 
+import Muse.UiComponents
+
 import Audacity.Effects
 
 import "../common"
 
 EffectBase {
 
-    property string title: reverb.title
+
     property alias instanceId: reverb.instanceId
 
-    width: 300
-    height: 200
+    property string title: qsTrc("effects", "Reverb")
+    property bool isApplyAllowed: true
+
+    height: 400
+    width: 400
 
     ReverbViewModel {
         id: reverb
     }
 
+    Component.onCompleted: {
+        reverb.init()
+    }
+
+    component ParamItem: Item {
+        id: param
+        property alias title: label.text
+        property real value: 0.0
+        property alias min: slider.from
+        property alias max: slider.to
 
+        height: 40
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.margins: 8
+
+        StyledTextLabel {
+            id: label
+            anchors.left: parent.left
+            anchors.top: parent.top
+            anchors.bottom: parent.bottom
+            verticalAlignment: Text.AlignVCenter
+            horizontalAlignment: Text.AlignLeft
+            width: 120
+        }
+
+        IncrementalPropertyControl {
+            id: numField
+            anchors.left: label.right
+            anchors.verticalCenter: parent.verticalCenter
+            width: 60
+            currentValue: param.value
```

Comment:
I'm missing something: `param` is the `ParamItem` instance, but I don't see it declare a member named `value`.

Reply:
here https://github.com/audacity/audacity/pull/7599/files#diff-d51d9585bf2291f98ec88a8b021e86c9cff150b62f06ef29c8a169804ac6e953R31

Reply:
Oy. Must have forgotten to ctrl+f...